### ISSUE TITLE
Delete history and open tabs on reboot

### ISSUE THREAD

**Issue by pfrancks** [GrapheneOS/Vanadium#342]
It would be great if one has a "semi always incognito mode" where browser history + data is deleted upon rebooting graphene os.

**Comment by dmint789** [comment]
Yes, I'd love this feature too. It should work like the clear history on close feature on desktop firefox.

**Comment by nerdguy1338** [comment]
As long as it's off by default, this sounds good.
